src/modules/compliance/src/lib/procedures/ensureFilePermissions.cpp

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#include <Evaluator.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <string.h>
+#include <unistd.h>
+#include <grp.h>
+#include <pwd.h>
+
+
+namespace compliance {
+
+AUDIT_FN(ensureFilePermissions)
```

Comment:
Could we get rid of these macros? it reads as 
```suggestion
Result<bool> Audit_ensureFilePermissions(std::map<std::string, std::string> args, std::ostringstream &logstream)
```

Reply:
Easier to generate the map + guarantees the consistency